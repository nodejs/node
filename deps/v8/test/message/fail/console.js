// Copyright 2017 the V8 project authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

console.time();
console.timeEnd();

console.time("abcd");
console.timeEnd({ toString: () => "ab" + "cd" });

console.timeLog('a');
console.timeEnd('a');
console.time("a");
console.timeEnd('a');

console.time("a", "b");
console.timeEnd("a", "b");

console.time('b');
console.time('b');
console.timeEnd('b');

console.time('ab');
console.timeLog('ab', {x: 1, y: 2});
console.timeEnd('ab');

console.timeStamp('stamp it baby!')

console.log("log", "more");
console.warn("warn", { toString: () => 2 });
console.debug("debug");
console.info("info");

console.info({ toString: () => {throw new Error("exception");} })
