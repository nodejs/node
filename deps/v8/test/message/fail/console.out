console.timeEnd: default, {NUMBER}
console.timeEnd: abcd, {NUMBER}
console.timeLog: Timer 'a' does not exist
console.timeEnd: Timer 'a' does not exist
console.timeEnd: a, {NUMBER}
console.timeEnd: a, {NUMBER}
console.time: Timer 'b' already exists
console.timeEnd: b, {NUMBER}
console.timeLog: ab, {NUMBER}
console.timeEnd: ab, {NUMBER}
console.timeStamp: stamp it baby!, {NUMBER}
log more
console.warn: warn 2
console.debug: debug
console.info: info
console.info: *%(basename)s:34: Error: exception
console.info({ toString: () => {throw new Error("exception");} })
                                ^
Error: exception
    at Object.toString (*%(basename)s:34:39)
    at console.info (<anonymous>)
    at *%(basename)s:34:9

